spec/components/provider_interface/application_timeline_component_spec.rb

```diff
@@ -118,6 +118,48 @@ def provider_user
     end
   end
 
+  context 'for an interview event' do
+    it 'renders the interview set up event' do
+      application_choice = create(:application_choice, status: 'interviewing')
+      interview = create(:interview)
+      application_choice_audit = create(:application_choice_audit, application_choice: application_choice)
+      application_choice_audit.update(audited_changes: { status: %w[awaiting_provider_decision interviewing] })
+      create(:interview_audit, interview: interview, application_choice: application_choice)
+      rendered = render_inline(described_class.new(application_choice: application_choice))
+      expect(rendered.text).to include 'Interview set up'
+      expect(rendered.text).to include '11 February 2020 at 10:00pm'
+      expect(rendered.css('a').text).to eq 'View interview'
+      expect(rendered.css('a').attr('href').value).to eq "/provider/applications/#{application_choice.id}/interviews#interview-#{interview.id}"
+    end
+
+    it 'renders the interview updated event' do
+      application_choice = create(:application_choice, status: 'interviewing')
+      interview = create(:interview)
+      application_choice_audit = create(:application_choice_audit, application_choice: application_choice)
```

Comment:
Can se use `build_stubbed` and method stubs so we can avoid having to create the application_choice in this instance. As we are already testing the audit retrieval as part of the query service specs I think we can skip it here.